ironfish/src/workerPool/tasks/submitTelemetry.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Metric } from '../../telemetry/interfaces/metric'
+import { WebApi } from '../../webApi'
+
+export type SubmitTelemetryRequest = {
+  type: 'submitTelemetry'
+  points: Metric[]
+}
+
+export type SubmitTelemetryResponse = {
+  type: 'submitTelemetry'
+}
+
+export async function submitTelemetry({
+  points,
+}: SubmitTelemetryRequest): Promise<SubmitTelemetryResponse> {
+  const api = new WebApi()
```

Comment:
I just noticed we don't consistency configure this from somewhere to let us test against staging, or let other users point at their own endpoint.